ocfweb/docs/docs/services/vhost.md

```diff
@@ -135,9 +135,17 @@ does not process nor advise off-site hosting requests, which can be
 [submitted directly to the university][offsite] (however, feel free to keep
 us informed if you have an existing OCF account).
 
-In other words, to comply with university policy, the OCF may only provide
+   In other words, to comply with university policy, the OCF may only provide
```

Comment:
Logically this paragraph belongs to the above bullet point, which is indented. Without the space, this paragraph is not indented. With only one bullet point, it was fine, but with two, it makes sense to promote it.